cockpit/leapp.html

```diff
@@ -141,10 +141,10 @@
 
         function call_leapp(cmd_args) {
             return cockpit.spawn(
-                ["/opt/leapp/bin/leapp-tool"].concat(cmd_args),
+                ["/usr/bin/leapp-tool"].concat(cmd_args),
```

Comment:
you're right and we'll have to eventually solve this - Probably through environment variables as you suggested - I have created issue #114 for it